### ISSUE TITLE
pip install missing dependency pbr

### ISSUE THREAD

**Issue by wreed4** [pycontribs/jira#501]
Steps to reproduce: 
```bash
$ pip install jira
...
Downloading jira-1.0.11-py2.py3-none-any.whl
...
$ python3
Python 3.6.3 (default, Dec 15 2017, 09:54:30)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.38)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
(ins)>>> from jira import Issue
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/private/tmp/sfng-build/.pyenv/versions/3.6.3/envs/sfng-build-/lib/python3.6/site-packages/jira/__init__.py", line 4, in <module>
    from pbr.version import VersionInfo
ModuleNotFoundError: No module named 'pbr'
(ins)>>> import jira
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/private/tmp/sfng-build/.pyenv/versions/3.6.3/envs/sfng-build-/lib/python3.6/site-packages/jira/__init__.py", line 4, in <module>
    from pbr.version import VersionInfo
ModuleNotFoundError: No module named 'pbr'
(ins)>>>
```

**Comment by Achimh3011** [comment]
Sorry, shot too fast.

I had the same problem, but it was just that I used a very old release of this project.

**Comment by Stealthii** [comment]
I know this wasn't intentional, but there needs to be better unit tests around simple things like import validation before considering releases. Unfortunately as this is a patch version release, it's likely breaking any software that relies on it.

I've lodged a PR #504 to resolve this issue, and would recommend a rapid release thereafter.

**Comment by hdost** [comment]
This has been fixed as of 1.0.11

**Comment by Stealthii** [comment]
@hdost this has not been fixed - 1.0.11 is the version with the bug. A new release is needed to fix the bug.

1.0.11 should ideally be removed from the PyPi repository, as it is a broken release.

**Comment by hdost** [comment]
@ssbarnea  This is still an issue, probably be best to release a 1.0.12

**Comment by GenPage** [comment]
@ssbarnea @hdost Any timeline on when 1.0.12 could be released or is this something that will need us to downgrade back to 1.0.10 for the time being?

**Comment by Stealthii** [comment]
Sorry to beat on about it, but I strongly recommend removing the 1.0.11 release from PyPi as soon as humanly possible, regardless of the timeline of a 1.0.12 release. Leaving broken packages on repo leads to all sorts of breakages across the landscape, and hurts the package's reputation.

**Comment by arthurspa** [comment]
Please, remove version 1.0.11 from PyPi.

**Comment by ssbarnea** [comment]
I made 1.0.11 on pypi and released 1.0.12, please test it ASAP before I made push the changes.

Be warned that I had to to something very ugly: edit git history. This is because some time ago we got an commit merged in Asian language which broke our packaging (due to ChangeLog generation). The only way to fix it was to fix this commit message. I didn't push my fix to gihub yet as I want a confirmation that 1.0.12 look ok (includes the pbr fix).

**Comment by ssbarnea** [comment]
Fixed in 1.0.14, which was also released few moments ago.

**Comment by blueskea** [comment]
I solve it by "sudo pip install pbr --ignore-installed"
